logger/options_test.go

```diff
@@ -31,6 +32,7 @@ func TestOptions_Validate(t *testing.T) {
 			if tc.valid {
 				assert.Equal(t, nil, err)
 			} else {
+				fmt.Println(err)
```

Comment:
we shouldn't need this here, is this just for testing?

Reply:
yeah so the logic here never gets invoked unless I have a fmt.Println- and without this statement there's a drop in coverage and coverall complains: 

https://github.com/logdna/logdna-go/pull/14/files#diff-c3644afec000d3b6093a29aab386824aR53

Reply:
In that case we should be testing the error output, not just triggering the code so that we get coverage on it.

`testify` has [assertions](https://godoc.org/github.com/stretchr/testify/assert#Assertions.EqualError) that should help here for validating that the error string matches

```golang
assert.EqualError(err,  "...")
```

Reply:
here's an modified version of this test as an example of what that would look like: https://gist.github.com/mdeltito/b38914612f9771487988710e60455575

an additional test that checks for multiple errors would be p cool too :wink: